### PR TITLE
Bug custom fields script

### PR DESCRIPTION
**Pull request by shakalee14**

### CODE REVIEW

**Comment by shakalee14** [comment]
- how to test this --> 
- create a campaign and upload contacts csv with custom fields
- create a script including custom fields
- assign a texter to the campaign and then switch to the texter view to see fields correctly populate

**Comment by sandramchung** [comment]
Tested branch as described above and it works.  👍

File: src/lib/scripts.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ const getScriptFieldValue = (contact, texter, fieldName) => {
   } else if (TOP_LEVEL_UPLOAD_FIELDS.indexOf(fieldName) !== -1) {
     result = contact[fieldName]
   } else {
-    result = contact.customFields[fieldName]
+    let customFieldNames = JSON.parse(contact.customFields)
```
Comment (sandramchung) [inline src/lib/scripts.js]: Nitpick: I think this this could be a const instead of let.

**Comment by sandramchung** [comment]
LGTM

**Pull request merged**